sycl/doc/GettingStartedXilinxFPGA.md

```diff
@@ -915,7 +915,7 @@ treating compiler warnings as errors and producing a compiler database
 to be used by tools like LSP server like `clangd`:
 
 ```bash
-python3 $SYCL_HOME/llvm/buildbot/configure.py --cuda \
+python3 $SYCL_HOME/llvm/buildbot/configure.py --xrt --cuda \
```

Comment:
```suggestion
python3 $SYCL_HOME/llvm/buildbot/configure.py --xrt --cuda --shared-libs \
```
shared libs greatly speed up the build so Ithink it should be added by default

Reply:
Good point! I thought it was the case by default.